app/views/users/piv_cac_recommended/show.html.erb

```diff
@@ -0,0 +1,33 @@
+<% self.title = t('titles.piv_cac_setup.upsell') %>
+
+<%= render AlertIconComponent.new(icon_name: :info_question) %>
```

Comment:
Some of the spacing in this page looks a little off from the design. We could use `StatusPageComponent` to help with this, though also recommending inline:

```suggestion
<%= render AlertIconComponent.new(icon_name: :info_question, class: 'margin-bottom-4') %>
```